app/server/lib/DocApi.ts

```diff
@@ -548,6 +548,25 @@ export class DocWorkerApi {
         .send(fileData);
     }));
 
+    // Starts transferring all attachments to the named store, if it exists.
+    this._app.post('/api/docs/:docId/attachments/transferAll', isOwner, withDoc(async (activeDoc, req, res) => {
+      await activeDoc.startTransferringAllAttachmentsToDefaultStore();
+      const locationSummary = await activeDoc.attachmentLocationSummary();
+
+      // Respond with the current status to allow for immediate UI updates.
+      res.json({
+        status: activeDoc.attachmentTransferStatus(),
+        locationSummary,
+      });
+    }));
+
+    // Returns the status of any current / pending attachment transfers
+    this._app.get('/api/docs/:docId/attachments/transferStatus', isOwner, withDoc(async (activeDoc, req, res) => {
```

Comment:
Take a look at the endpoint registered above (line 521), it will be matched first and this handler won't be executed.